### PR TITLE
glaksmono mini project

### PR DESCRIPTION
**Pull request by glaksmono**

### CODE REVIEW

File: app/Repositories/UserRepository.php [hunk 1]
```diff
@@ -112,6 +113,23 @@ public function counts()
     }
 
     /**
+     * Get blog authors, their number of blogs, the last created date, and title of their latest blog entry.
+     *
+     * @return Illuminate\Database\Eloquent\Builder
+     */
+    public function getAuthorPostsWithOrder()
```
Comment (aphofstede) [inline app/Repositories/UserRepository.php:120]: This method name suggests to me it returns one author's posts in a certain order

Comment (glaksmono) [reply app/Repositories/UserRepository.php:120]: It's definitely `ASC` order, was just thinking `AuthorsPosts` doesn't sound nice haha

File: app/Repositories/UserRepository.php [hunk 2]
```diff
@@ -112,6 +113,23 @@ public function counts()
...
+    {
+        $authors = $this->model
+            ->join('posts', 'posts.user_id', '=', 'users.id')
+            ->select(DB::raw('count(posts.id) as total_count'), 'username', 'posts.id', 'posts.created_at', 'posts.title')
```
Comment (aphofstede) [inline app/Repositories/UserRepository.php:124]: How did you come by DB::raw()?

Comment (aphofstede) [reply app/Repositories/UserRepository.php:124]: I see.. https://blog.jetbrains.com/phpstorm/2015/01/laravel-development-using-phpstorm/ IntelliJ needs some help for Laravel facades..

Comment (glaksmono) [reply app/Repositories/UserRepository.php:124]: Ah, I added this tho: https://github.com/aphofstede/laravel5-3-example/pull/1/files#diff-7148cca22aa02ecc66532f4c02d2f3f7R7

Comment (aphofstede) [reply app/Repositories/UserRepository.php:124]: Facades are weird...

**Pull request closed without merging**